src/Utilities/Compiler/Extensions/IOperationExtensions.cs

```diff
@@ -29,9 +29,19 @@ internal static partial class IOperationExtensions
         {
             if (invocation.Instance != null)
             {
-                return beforeConversion ?
-                    GetReceiverType(invocation.Instance.Syntax, compilation, cancellationToken) :
-                    invocation.Instance.Type as INamedTypeSymbol;
+                if (beforeConversion)
+                {
+                    if (invocation is { Parent: IConditionalAccessOperation conditionalAccess })
```

Comment:
Again, required due to VB operation tree

Reply:
Can you clarify the difference between C# and VB?

Reply:
This VB compiler test

https://github.com/dotnet/roslyn/blob/aaee0047096782110b43a87df7d0e85e14b1af68/src/Compilers/VisualBasic/Test/IOperation/IOperation/IOperationTests_IConditionalAccessExpression.vb#L28-L35

shows that `conditionalAccess.Operation.Syntax` is the same as `invocation.Instance.Syntax`. Is there something I'm missing?

Reply:
In C# the `invocation.Instance` is null, whears in VB it as a `IConversionOperation`

Reply:
Please add a comment in code on the same lines.